### DISCUSSION TITLE
Cannot create container for service worker-service

### DISCUSSION THREAD

**Discussion by mshiner** [Budibase/budibase#5827] [Help] [answered]
Hi using the instructions from the  [docs](https://docs.budibase.com/docs/docker-compose) to self host on an ubuntu 2204 server.

I created a budibase directory, copied the files and issued the docker-compose up command.

The following errors were generated:

```
Creating bbworker                      ... error

ERROR: for bbworker  Cannot create container for service worker-service: No such image: budibase.docker.scarf.sh/budibase/worker: image not known

ERROR: for worker-service  Cannot create container for service worker-service: No such image: budibase.docker.scarf.sh/budibase/worker: image not known
ERROR: Encountered errors while bringing up the project.
```

Any help appreciated.

Thanks
M

**Comment by aderici** [comment]
I believe they are using a proxy service but you should be fine if you change image name to :  

budibase/worker 

instead of 

budibase.docker.scarf.sh/budibase/worker

**Reply by shogunpurple** [reply]
Yep - it seems as though somehow your network configuration is not picking up the scarf domain. I just tried to pull that exact image and it worked as expected - if you change to `budibase/worker`, “you will likely need to do this for `budibase/apps` as well, for the app service.

**Reply by mshiner** [reply]
Thanks both for the timely response and insight.

I tried your suggestions but still get the same error. The system is running Ubuntu 2204 with Podman instead of Docker - this normally works OK with docker-compose YAML files but not this time for some reason.

I also tried adding "budibase.docker.scarf.sh" to /etc/containers/registries.conf. I think I'll fire up a new 2204 server with just Docker and not Podman and see how I get on.

Thanks

M

**Comment by mshiner** [comment, marked as answer]
Fired up a new 2204 vm and started from scratch

All sorted now!!!

Thanks again.

M
